roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -168,7 +168,7 @@ def is_available_skopeo_image(self, image, default_registries):
             registries = [registry]
 
         for registry in registries:
-            args = {"_raw_params": "skopeo inspect --tls-verify=false docker://{}/{}".format(registry, image)}
+            args = {"_raw_params": "timeout 10 skopeo inspect --tls-verify=false docker://{}/{}".format(registry, image)}
```

Comment:
would have been nice if a `timeout` flag was supported directly in the [inspect subcommand](https://github.com/projectatomic/skopeo/blob/master/cmd/skopeo/inspect.go#L44) :)

Reply:
Yeah. Or in the ansible command module. Alas.
However "timeout" comes from coreutils and does the job fine.